### PR TITLE
Issue #13345: Enable HideUtilityClassConstructorCheckExamplesTest

### PR DESCRIPTION
**Pull request by andreiyusupau**
Issue: https://github.com/checkstyle/checkstyle/issues/13345
Enabled HideUtilityClassConstructorCheckExamplesTest.
Replaced its Example.txt with Example.java.
Added expected violations to test.

### CODE REVIEW

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/Example1.java [hunk 1]
```diff
@@ -9,31 +9,31 @@
 // xdoc section -- start
 class Test { // violation, class only has a static method and a constructor
 
-  public Test() {
-  }
+    public Test() {
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/Example1.java:12]: Please do not close PRs.
Please lean how to reuse PRs.

Indentation should be +2

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/Example1.java (hunk 1 above)
Comment (andreiyusupau) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/Example1.java]: PR closed automatically, I think it happened when I force pushed to fork's master.
Fixed identation

Comment (nrmancuso) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/Example1.java]: @andreiyusupau always allow reviewers to resolve conversations

**Comment by romani** [comment]
Ply do not do development in master branch.
Please read https://checkstyle.org/beginning_development.html#Starting_Development

You need to close this PR and start from scratch.

**Pull request closed without merging**